honorary-one-line-wonders/c/helloWorld.c

```diff
@@ -0,0 +1,6 @@
+#include <stdio.h>
+int main()
+{
+   printf("Hello, World!");
```

Comment:
I think this actually qualifies as a one-line-wonder since the print statement is one line.